CMSgov/design-system
# [WNMGDS-2984] Update Pagination Styling for Medicare

## Summary

- [Ticket](https://jira.cms.gov/browse/WNMGDS-2984)
- Changes:
- Added an override to the base.scss for the medicare child design system
- Should set the button font family to Montserrat for the Pagination elements.

## How to test

1. Fire it up locally and confirm in Storybook/Docsite that the buttons in the Pagination component are indeed using the font family Montserrat
2. Or check out this [demo branch](https://cmsgov.github.io/design-system/branch/jryan/wnmgds-2984-add-in-new-pagination-styling)

## Checklist
